taskchampion/xtask/src/main.rs

```diff
@@ -3,29 +3,123 @@
 //! At the moment it is very simple, but if this grows more subcommands then
 //! it will be sensible to use `clap` or another similar library.
 
+use regex::Regex;
 use std::env;
 use std::fs::File;
-use std::io::Write;
-use std::path::PathBuf;
+use std::io::{BufRead, BufReader};
+use std::io::{Seek, Write};
+use std::path::{Path, PathBuf};
+
+/// The const MSRV_PATHS_REGEX in /xtask/main.rs is an array of tuples
+///  of the form (PATH, REGEX) where PATH and REGEX are literals where
+///  PATH is a file that updates the Minimum Supported Rust Version
+///  and REGEX is the pattern to find the appropriate line in the file
+
+// Increment length of array when adding tuples of (PATH, REGEX).
```

Comment:
Let's make this all a single docstring, instead of referring to the identifier by name and filename.

```suggestion
/// Tuples of the form (PATH, REGEX) where PATH and REGEX are literals where PATH is a file that
/// conains the Minimum Supported Rust Version and REGEX is the pattern to find the appropriate
/// line in the file. PATH is relative to the `taskchampion/` directory in the repo.
```